### ISSUE TITLE
Disable TLS 1.0

### ISSUE THREAD

**Issue by ulc-brian** [tlaverdure/laravel-echo-server#305]
With the laravel echo json file I would like to pass an option:

"secureOptions": constants.SSL_OP_NO_TLSv1

Which would allow the server to startup with TLS 1.0 disabled. The problem, is since these options are specified in the json file, that constants string needs to be in quotes to be valid json. When your server parses the json file it simply passes it in as that string instead of converting it to utilize the constant needed.

Would love to have support for disabling TLS 1.0 or a way to utilize constants so that any option that needs a constant can be supported.

https://stackoverflow.com/questions/31201029/how-to-disable-tls-1-0-and-use-only-tls-1-1-using-nodejs

**Comment by ulc-brian** [comment]
By the way this was needed for PCI-DSS compliance due to the fact that TLS 1.0 must be disabled by June 30, 2018. Until a feature like this is implemented, I managed to find a work around. I created a simple node script:

```
const { constants } = require('crypto')
console.log(constants.SSL_OP_NO_TLSv1);   
```

Which translates the constant I needed into a number. I tested on multiple servers and that constant is always equal to:

```
67108864
```

This is a hacky solution for now, but for anybody else needing this could be helpful. You would add this to your laravel echo json file:

```
{
	"authHost": "https://yourwebsite.com",
	"authEndpoint": "/broadcasting/auth",
	...
	"secureOptions": 67108864
}
```

That would effectively disable TLS 1.0. The better plan, would be either to create a wrapper around laravel echo server, or see if the latest version of NodeJS has TLS 1.0 off by default.
